src/components/header/style.module.css

```diff
@@ -483,7 +483,7 @@
 
 :global(.aa-dropdown-menu) {
 	position: fixed !important;
-	top: 60px !important;
+	top: var(--header-height) !important;
```

Comment:
`--header-height` is `3.5.rem`/`56px` by default, which 60px is pretty close to, but any level of zoom and it'll start overlaying incorrectly.